### ISSUE TITLE
[ Feature  Request] Add Animation when cursor hides

### ISSUE THREAD

**Issue by Permafrozen** [VirtCode/hypr-dynamic-cursors#39]
There is a option for hyprland, where the cursor hides after a set time. But it just disappears, really boring.

I would propose adding a fading animation or something similar. Maybe the cursor shrinking or something like that.

**Comment by VirtCode** [comment]
True, that's actually a really cool idea. Thanks for the suggestion!

**Comment by jaredmontoya** [comment]
ideas:
- fade
- shrink until disappears
- explode with confetti

**Comment by 0x000C0A71** [comment]
If you could combine this with #34 that would be great :+1:
